src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -306,6 +314,43 @@ public override async Task RunAsync()
             return result;
         }
 
+        private async Task DotnetIsolatedLinuxValidation(Site functionApp)
+        {
+            Dictionary<string, string> updatedSettings = null;
+            if (functionApp.IsDynamic)
+            {
+                if (!string.IsNullOrEmpty(functionApp.LinuxFxVersion))
+                {
+                    updatedSettings = new Dictionary<string, string>
+                    {
+                        [Constants.LinuxFxVersion] = ""
+                    };
+                }
+            }
```

Comment:
Hmm, even if this is Linux Consumption, we should still set it to "DOTNET-ISOLATED|5.0" right? This should be similar to "PYTHON|3.6", which will use the default mesh image.

Reply:
It unfortunately doesn't work right now. Linux consumption blocks setting it to unknown values and it doesn't know this value. The fix is merged in the platform, but it will take some time for it to be rolled.